src/docs/guides/style-guide.md

```diff
@@ -14,7 +14,7 @@ contributors:
 
 This is a component style guide created and enforced internally by the core team of Stencil, for the purpose of standardizing Stencil components. This should only be used as a reference for other teams in creating their own style guides. Feel free to modify to your team's own preference.
 
-> In order to enforce this (or your team's) style guide, we recommend leveraging a static analysis tool like [ESLint](https://eslint.org/). [@stencil/eslint-plugin](https://www.npmjs.com/package/@stencil/eslint-plugin) provides rules specifically for writing Stencil components.
+> In order to enforce this (or your team's) style guide, we recommend leveraging a static analysis tool like [ESLint](https://eslint.org/). [@stencil-community/eslint-plugin](https://www.npmjs.com/package/@stencil-community/eslint-plugin) provides rules specifically for writing Stencil components.
```

Comment:
It looks like the package hasn't been published under the `@stencil-community` scope yet, so right now the link (https://www.npmjs.com/package/@stencil-community/eslint-plugin) asks me to sign in - do we need to wait to merge this until the package has been published for the first time?

Reply:
Yup, exactly. I shoulda spelled that out here, but we'll end up getting these reviewed, then move the repo over to the community, then merge the commit to `stencil-eslint`, then publish the package under the correct scope, then land this.

> asks me to sign in

Classic npm. I think it thinks you're trying to access a private package and assumes you need to sign in. I _think_

Reply:
Cool, that all makes sense to me, figured it was something like that!